### ISSUE TITLE
Unknown error in post to my notifications

### ISSUE THREAD

**Issue by Rohan3479** [trunks-social/trunks-public#321]
Got a notification. "Unknown event. Report this to trunks.social". I can't interact with the notification in any way. Screen shot is attached. 
![20241217_152740](https://github.com/user-attachments/assets/664af0fd-778a-4a98-be14-184373a15e35)

**Comment by Decad3nce** [comment]
Yeah I think this was the recap from Mastodon. I'll see if I can support it somehow.
